explainers/DocumentationSearch/explainer.md

```diff
@@ -0,0 +1,68 @@
+# DevTools: Documentation Search
+Authors:
+- [Erica Draud](https://github.com/erdraud), Program Manager
+- Sean Larkin
```

Comment:
@TheLarkInn I used a lot of text from your document. I can keep your name and add your github account or remove your name. Let me know what you prefer.

Reply:
Whatever you prefer is fine. I don't mind either :+1: